### ISSUE TITLE
clean up github teams and organization members

### ISSUE THREAD

**Issue by naved001** [CCI-MOC/ops-issues#1302]
we'll do this as part of spring cleanup!

**Comment by tssala23** [comment]
@joachimweyl Lars kinda did this with the new github config repo no? https://github.com/OCP-on-NERC/github-config

**Comment by joachimweyl** [comment]
@naved001 do the changes @larsks made close this issue?

**Comment by naved001** [comment]
@joachimweyl No, Lars did the cleanup of a different organization (OCP-on-NERC). The CCI-MOC repository has bunch of different teams that include PhD students, professors other researchers.
